problem_builder/answer.py

```diff
@@ -115,6 +115,20 @@ def add_error(msg):
         if not data.name:
             add_error(u"A Question ID is required.")
 
+    def student_view_user_state(self, context=None):
```

Comment:
This needs to be an `@XBlock.handler` (but not `json_handler` since that only works with POST requests)